securedrop/source_app/main.py

```diff
@@ -72,8 +72,18 @@ def create():
             del session['codenames']
 
             filesystem_id = current_app.crypto_util.hash_codename(codename)
+            try:
+                source = Source(filesystem_id, current_app.crypto_util.display_id())
+            except ValueError as e:
+                current_app.logger.error(e)
+                flash(
+                    gettext("There was a temporary problem creating you account. "
```

Comment:
typo: you->your

Reply:
Oops. Thanks for pointing that out. Corrected it.